src/config/section/storage.js

```diff
@@ -101,6 +101,26 @@ export default {
           label: 'label.action.attach.disk',
           message: 'message.confirm.attach.disk',
           args: ['virtualmachineid'],
+          mapping: {
```

Comment:
If we were to only show Vms in stopped/running state, then please create a new component